libexec/pyenv-init

```diff
@@ -112,8 +114,15 @@ function detect_profile() {
     rc='~/.profile'
     ;;
   * )
-    profile='your shell'\''s login startup file'
-    rc='your shell'\''s interactive startup file'
+    # if no_config_place_holder is set,
+    # do not filled in any placehodler for config file
+    if [ -n "$no_config_place_holder" ]; then
```

Comment:
This should not be a global variable.

Either it should be an optional argument to the function, or an exit code should indicate whether shell detection was successful.

Reply:
b502c9a9dffb6be593e189459e3cbacb8b15d960